GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,162 @@
+
+```
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
+```
+Now create a Dockerfile inside the `kogito-examples/process-business-rules-quarkus`
+
+With the following content:
+
+```
+FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
+
+COPY target/*-runner.jar $KOGITO_HOME/bin    
+COPY target/lib $KOGITO_HOME/bin/lib
+```
+
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+
+Now let’s push the above built image to a public container registry.
+
+`$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest`
+
+## Install the Operator
+First we would need to enable the OLM in our Minikube cluster for that just run:
+
+`$ minikube addons enable olm`
+
+Launch the OLM console locally, 
+Just clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repo and from the root of the project run: `$ make run-console-local`.
+ 
+ This will run the operatorhub console on http://localhost:9000 
+
+Note: Needs to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where kogito-operator and all the dependent operator will run
+
+`$ kubectl create ns kogito`
+
+Now on your browser and visit  https://localhost:9000. Select `Operators > OperatorHub` and search for kogito. 
+Select the kogito operator by Redhat and install it with defaults only changing the namespace where it needs to be installed. You can select the `kogito` namespace which was created for this purpose.
+
+You can see the pods by:
+
+`$ kubectl get po -n kogito`
+
+Wait till all pods are in running state (It is installing kogito-operator and all the dependent operator for kogito)
+
+
+Alternatively you if you are having trouble in having the operators shown in the OperatorHub.
+You can also install it from [here](https://operatorhub.io/operator/kogito-operator)
+
+## Run Kogito Service with Kogito Operator
+
+Now we’ll create a `KogitoRuntime` object with the Application image that we built earlier.
+
+```
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoRuntime
+metadata:
+  name: process-business-rules-quarkus
+spec:
+  replicas: 1
+  image:
+    namespace: <your-quay-namespace>
+```
+Create a myapp.yml with the above content.
+
+Just create the above object with
+
+`$ kubectl create -f myapp.yml -n <operator-namespace>`
+
+You can see your runtime object by 
+
+`$ kubectl get KogitoRuntime -n <operator-namespace>`
+
+Now expose the service on NodePort so it’s easily accessible.
+
+`$ kubectl expose deployment process-business-rules-quarkus -n <operator-namespace>  --type=NodePort --name=process-business-rules-quarkus`
+
+`$ minikube service process-business-rules-quarkus -n <operator-namspace>`
+
```

Comment:
Maybe adding the output, so users will see the location of the exposed service?

Reply:
The above command will open the service in your default browser, do you want to me add a screenshot with that?

Reply:
No need! Yes, you're right! :)